jupiterone/modifiers.go

```diff
@@ -355,15 +355,7 @@ func (v jsonValidator) ValidateString(ctx context.Context, req validator.StringR
 // ValidateList implements validator.List
 func (v jsonValidator) ValidateList(ctx context.Context, req validator.ListRequest, resp *validator.ListResponse) {
 	var vals []string
-	err := req.ConfigValue.ElementsAs(ctx, &vals, false)
-	if err != nil {
-		resp.Diagnostics.Append(validatordiag.InvalidAttributeValueDiagnostic(
-			req.Path,
-			"not a valid string: "+v.Description(ctx),
-			req.ConfigValue.String(),
-		))
-		return
-	}
+	req.ConfigValue.ElementsAs(ctx, &vals, false)
```

Comment:
By ignoring the error return here, we are potentially missing valid errors. Also, if this does return an error, then `vals` is an empty string so none of the logic below this line is validated